### ISSUE TITLE
Can't get the result as described

### ISSUE THREAD

**Issue by wtwong316** [l294265421/chat-sentiment-analysis#1]
I have followed the steps to run.
When the last step is run, I can open a browse for the 7680 port.
But I cannot get the result of the support tasks as shown. 
Would you mind show how to achieve the result?
Thanks.

**Comment by l294265421** [comment]
> I have followed the steps to run. When the last step is run, I can open a browse for the 7680 port. But I cannot get the result of the support tasks as shown. Would you mind show how to achieve the result? Thanks.

The bootstrap command in the original README is wrong. Try this one:
nohup sh run.sh chat_sentiment_analysis/llama/inference_alpaca_lora_gradio.py > inference_alpaca_lora_gradio.log 2>&1 &

A demo of this repository has been deployed: https://ff280230d6f28b297a.gradio.live/. An example is showed bellow:
<img width="1518" alt="图片" src="https://user-images.githubusercontent.com/9948265/233820269-3408d0cb-f134-41f7-a4ab-644466a9ec01.png">

**Comment by l294265421** [comment]
> I have no luck. As attached, the result shown is not the same as you posted. I try to demo, the result is the same you posted. Since I really want your work to be my basis to develop, would you mind downloading your codes, performing the step and rerun the command to see whether it works. ![Screenshot_2023-04-24_22-27-34](https://user-images.githubusercontent.com/8217228/234027156-0a3f0ad3-d18d-4010-8a62-4d2c0746a62e.png)

It is ok to download the code of this repo. It seems that the model you load is the orginal LLaMA. Please make suer that you load the  trained model correctly.

**Comment by wtwong316** [comment]
I have run all the four steps as suggested.
What else I need to do?
Or you can show me where is the trained model and what the size is?
-----------------
Step by Step
nohup sh run.sh chat_sentiment_analysis/llama/finetune.py > autodl.log 2>&1 &
sh run.sh chat_sentiment_analysis/llama/inference_llama.py
sh run.sh chat_sentiment_analysis/llama/inference_alpaca_lora.py
nohup sh run.sh chat_sentiment_analysis/llama/inference_alpaca_lora_gradio.py > inference_alpaca_lora_gradio.log 2>&1 &
------------------------

**Comment by l294265421** [comment]
If you follow the steps to fine-tune and test the model, the correct model should be loaded. I am not sure what causes the problem. Maybe you can try the trained model and the original LLaMA to observe the difference and find the problem. Or observe the training loss and see whether the model is trained well. The trained lora weights are in the sub directory of the project directory:
<img width="1082" alt="图片" src="https://user-images.githubusercontent.com/9948265/234045125-3635e4eb-8109-449d-8f54-36f6a2b4641f.png">

**Comment by wtwong316** [comment]
![Screenshot_2023-04-24_23-54-47](https://user-images.githubusercontent.com/8217228/234050848-93bc4790-c1a8-4847-a763-fbd2cee955a0.png)

It seems my finetune did not produce the model correctly.

I have run the given command. 
nohup sh run.sh chat_sentiment_analysis/llama/finetune.py > autodl.log 2>&1 &
Please give me a hint if what'll cause the 'runs' file/directory  not existed.
Thanks.

**Comment by wtwong316** [comment]
I can see that there is a strange warning message.
Any hint from that message.
-----
The tokenizer class you load from this checkpoint is not the same type as the class this function is called from. It may result in unexpected tokenization.
The tokenizer class you load from this checkpoint is 'LLaMATokenizer'.
The class this function is called from is 'LlamaTokenizer'
---------------------------

**Comment by l294265421** [comment]
> ![Screenshot_2023-04-24_23-54-47](https://user-images.githubusercontent.com/8217228/234050848-93bc4790-c1a8-4847-a763-fbd2cee955a0.png)
> 
> It seems my finetune did not produce the model correctly.
> 
> I have run the given command. nohup sh run.sh chat_sentiment_analysis/llama/finetune.py > autodl.log 2>&1 & Please give me a hint if what'll cause the 'runs' file/directory not existed. Thanks.

Try to finetune again

**Comment by wtwong316** [comment]
I have cleaned the huggingface cache, regenerate the test_data.json and rerun. Still got no luck. Here is the main difference.

Yours> Downloading and preparing dataset json/default to /root/.cache/huggingface/datasets/json/default-915bb771586491d9/0.0.0...

Mine> Downloading and preparing dataset json/default to /home/wtwong/.cache/huggingface/datasets/json/default-6bc8178fec238819/0.0.0...

I have made some screenshot and please check anything missing (e.g. settings or parameters need to give to the finetune program).
![1](https://user-images.githubusercontent.com/8217228/234275764-af96a5eb-b5a6-4747-a827-32109a9da03c.png)

![2](https://user-images.githubusercontent.com/8217228/234275794-3f4809c3-f924-47b2-a9d0-850b6f35669a.png)

![3](https://user-images.githubusercontent.com/8217228/234275824-f89ec386-107b-46ba-8448-8740e6d85f59.png)

![4](https://user-images.githubusercontent.com/8217228/234275851-7ddc848f-660a-4eb0-b6ac-20deeba69b43.png)

![5](https://user-images.githubusercontent.com/8217228/234275898-c51aa6a0-cbfe-4379-abbc-77abd7de38b5.png)
